contracts/lib/constants.sol

```diff
@@ -4,6 +4,9 @@ pragma solidity 0.8.13;
 /// @dev The approximate number of blocks per year that is assumed by the interest rate model
 uint256 constant BLOCKS_PER_YEAR = 10_512_000;
 
+/// @dev The approximate number of seconds per year(considered 1 second = 1 block), it is assumed by the interest rate model
```

Comment:
I don't understand the second part of the comment: `(considered 1 second = 1 block), it is assumed by the interest rate model`